CONTRIBUTING.rst

```diff
@@ -83,9 +83,52 @@ Using a custom ``virtualenv``
     $ cd ${HOME}/hack-on-google-cloud-python
     $ ${VENV}/bin/python setup.py install
 
+- To keep your ``virtualenv`` from getting checked-in, you can make a
+  global gitignore file::
+
+  $ echo "${ENV_NAME}" >> ~/gitignore_global
+  $ git config --global core.excludesfile ~/.gitignore_global
+
   Unfortunately using ``setup.py develop`` is not possible with this
   project, because it uses `namespace packages`_.
 
+Using ``virtualenvwrapper``
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- To install ``google-cloud-python`` with ``virtualenvwrapper`` you
+  can run the following::
+
+  $ pip install virtualenvwrapper
+  $ export WORKON_HOME=~/Envs
+  $ source /usr/local/bin/virtualenvwrapper.sh
+
+  or for Windows::
+
+  $ pip install virtualenvwrapper-win
+
+- Then create your virtualenv::
+
+  $ mkvirtualenv venv
```

Comment:
So RST doesn't like the `-` lists and then code blocks it seems.

The [previous code above](https://github.com/GoogleCloudPlatform/google-cloud-python/pull/2501/files#diff-ea1f65a8eaac0cec7ae151dfdbbbc587R68) also suffers from the same highlighting, but I was basing it off of how it looks in the github preview.

I've updated with the alternate version that looks kind of weird on github preview.